src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
```

Comment:
Setting UTF-8 (or any encoding) by default could also create issues. Windows CMD uses bitmap fonts by default ("Raster Fonts"), which do **not** support unicode characters; A TrueType font must be set.

So if UTF-8 is the default, and a user runs NUnit from Windows CMD console with the default settings, they will see random characters in place of their text. Which I believe qualifies this as an opt-in condition.